scripts/install.sh

```diff
@@ -61,8 +61,13 @@ chmod a+rx /usr/local/bin
 
 systemctl reload ssh
 
-# hardcode /etc/hosts entries so we don't need DNS
+# disable cloud-init management of /etc/hosts
+if ! grep -q "manage_etc_hosts: false" /etc/cloud/cloud.cfg; then
```

Comment:
test-backend has overrides in `/etc/cloud/cloud.cfg.d/` - if TPP does as well we could do that & just write this direct into e.g. `/etc/cloud/cloud.cfg.d/999-opensafely`?

Reply:
interestingly, maybe we actually already have this set on test-backend in `/etc/cloud/cloud.cfg.d/90-digitalocean.cfg` & the contents of `/etc/hosts` is misleading?

Reply:
Just saw that

bah